code/obj/decorations.dm

```diff
@@ -1005,3 +1005,81 @@ obj/decoration/ceilingfan
 
 	examine()
 		return list()
+
+//fake guns for shooting range prefab
+
+/obj/item/gun/laser_pistol
+	name = "laser pistol"
+	icon = 'icons/obj/decoration.dmi'
+	desc = "A terribly cheap and discontinued old model of laser pistol."
+	icon_state = "laser_pistol"
+	inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+	item_state = "protopistol"
+	stamina_damage = 0
+	stamina_cost = 4
+	stamina_crit_chance = 0
+	throwforce = 0
+
+	attack_hand(mob/user as mob)
+		if ((user.r_hand == src || user.l_hand == src) && src.contents && length(src.contents))
+			user.visible_message("The cell on this is corroded. Good luck getting this thing to fire ever again!")
+			src.add_fingerprint(user)
+		else
+			return ..()
+
+/obj/item/gun/laser_pistol/prototype
+	name = "prototype laser pistol"
+	icon = 'icons/obj/decoration.dmi'
+	desc = "You've never heard of this pistol before...who made it?"
```

Comment:
```suggestion
	desc = "You've never heard of this pistol before... who made it?"
```
minor grammar

Reply:
fixed up all the grammar and spelling things! ty!